profiler/lib/runInstructionInWebDriver.ts

```diff
@@ -1,17 +1,17 @@
 import { By, Key, until, WebDriver, WebElement } from 'selenium-webdriver';
-import IDirective from '@double-agent/runner/interfaces/IDirective';
-import IDirectivePage from '@double-agent/runner/interfaces/IDirectivePage';
+import IInstruction from '@double-agent/runner/interfaces/IInstruction';
+import IInstructionPage from '@double-agent/runner/interfaces/IInstructionPage';
 
-export default async function runDirectiveInWebDriver(
+export default async function runInstructionInWebDriver(
   driver: WebDriver,
-  directive: IDirective,
+  instruction: IInstruction,
```

Comment:
The naming turns out to be a bit weird when it's a set of "instructions" per instruction, doesn't it? I'm now a bit thrown by the plural vs singular of this.

Reply:
BTW, I know this is one of those "I'm not sure what to do with this" comments. If you feel like this works, I'll see if it grows on me. Just realized as I was reading the code that this was tripping me up in a different way.

Reply:
When you say, " a set of instructions per instruction", are you referring to the pages array within the instruction?

Reply:
`Assignment` seems to work better. PR has been updated.